<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidgetTest</class>
 <widget class="QDockWidget" name="DockWidgetTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>339</width>
    <height>725</height>
   </size>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="windowTitle">
   <string>Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxSpeed">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Turn Rate [deg/s]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxTurn</cstring>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxClimb">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-650.000000000000000</double>
      </property>
      <property name="maximum">
       <double>650.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="19" column="1" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxPress">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>28.000000000000000</double>
      </property>
      <property name="maximum">
       <double>31.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Pressure [inHg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxPress</cstring>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxDevV">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxHead">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-360.000000000000000</double>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Vertical Deviation [-]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxDevV</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxAlpha">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Horizontal Deviation [-]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxDevH</cstring>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxDevH">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxMach">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="18" column="1" colspan="2">
     <widget class="QPushButton" name="pushButtonAuto">
      <property name="text">
       <string>AUTO</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxAlt">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Angle of Attack [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxAlpha</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Roll [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxRoll</cstring>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxADF">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-360.000000000000000</double>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Altitude [ft]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxAlt</cstring>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxDME">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Airspeed [kts]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxSpeed</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="17" column="1" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Mach Number [-]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxMach</cstring>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>DME distance [nm]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxDME</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxTurn">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-7.000000000000000</double>
      </property>
      <property name="maximum">
       <double>7.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Heading [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxHead</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxSlip">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Sideslip [-]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxSlip</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxPitch">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-90.000000000000000</double>
      </property>
      <property name="maximum">
       <double>90.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Angle of Sideslip [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxBeta</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Pitch [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxPitch</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxRoll">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Climb Rate [ft/min]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxClimb</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxBeta">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-15.000000000000000</double>
      </property>
      <property name="maximum">
       <double>15.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>ADF Bearing [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxADF</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Course [deg]:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>spinBoxCourse</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxCourse">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBoxAlpha</tabstop>
  <tabstop>spinBoxBeta</tabstop>
  <tabstop>spinBoxRoll</tabstop>
  <tabstop>spinBoxPitch</tabstop>
  <tabstop>spinBoxHead</tabstop>
  <tabstop>spinBoxSlip</tabstop>
  <tabstop>spinBoxTurn</tabstop>
  <tabstop>spinBoxCourse</tabstop>
  <tabstop>spinBoxDevH</tabstop>
  <tabstop>spinBoxDevV</tabstop>
  <tabstop>spinBoxSpeed</tabstop>
  <tabstop>spinBoxMach</tabstop>
  <tabstop>spinBoxAlt</tabstop>
  <tabstop>spinBoxPress</tabstop>
  <tabstop>spinBoxClimb</tabstop>
  <tabstop>spinBoxADF</tabstop>
  <tabstop>spinBoxDME</tabstop>
  <tabstop>pushButtonAuto</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
